lib/improver/cli/snow_falling_level.py

```diff
@@ -84,23 +84,74 @@ def main(argv=None):
                               "derived value."))
     args = parser.parse_args(args=argv)
 
+    # Load Cubes
     temperature = load_cube(args.temperature, no_lazy_load=True)
     relative_humidity = load_cube(args.relative_humidity, no_lazy_load=True)
     pressure = load_cube(args.pressure, no_lazy_load=True)
     orog = load_cube(args.orography, no_lazy_load=True)
     land_sea = load_cube(args.land_sea_mask, no_lazy_load=True)
 
-    result = FallingSnowLevel(
-        precision=args.precision,
-        falling_level_threshold=args.falling_level_threshold).process(
-            temperature,
-            relative_humidity,
-            pressure,
-            orog,
-            land_sea)
+    # Process Cube
+    result = process(temperature, relative_humidity, pressure, orog,
+                     land_sea, args.precision, args.falling_level_threshold)
 
+    # Save Cube
     save_netcdf(result, args.output_filepath)
 
 
+def process(temperature, relative_humidity, pressure, orog, land_sea,
+            precision=0.005, falling_level_threshold=90.0):
+    """Module to calculate continuous snow falling level.
+
+    Calculate the wet bulb temperature integral by firstly calculating the
+    wet bulb temperature from the inputs provided and then calculating the
+    vertical integral of the wet bulb temperature.
+    Find the falling_snow_level by finding the height above sea level
+    corresponding to the falling_level_threshold in the integral data.
+
+    Args:
+        temperature (iris.cube.Cube):
+            Cube of air temperature at heights (m) at the points for which the
+            continuous falling snow level is being calculated.
+        relative_humidity (iris.cube.Cube):
+            Cube of relative humidities at heights (m) at the points for which
+            the continuous falling snow level is being calculated.
+        pressure (iris.cube.Cube):
+            Cube of air pressure at heights (m) at the points for which the
+            continuous falling snow level is being calculated.
+        orog (iris.cube.Cube):
+            Cube of the orography height in m of the terrain over which the
+            continuous falling snow level is being calculated.
+        land_sea (iris.cube.Cube):
+            Cube containing the binary land-sea mask for the points for which
+            the continuous falling snow level is being calculated. Land points
+            are set to 1, sea points are set to 0.
+
+    Kwargs:
+        precision (float):
+            Precision to which the wet bulb temperature is required: This is
+            used by the Newton iteration.
+            Default is 0.005.
+        falling_level_threshold (float):
+            Cutoff threshold for the wet-bulb integral used to calculate the
```

Comment:
'wet-bulb' is hyphenated here, but is not elsewhere in the text. It should probably be hyphenated everywhere.